From aa874b6e80a7614c6f4c11726116dda9beec0a82 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Mon, 6 Jan 2025 15:09:30 +0100
Subject: [PATCH] Revert "arm64: dts: st: enable CoreSight on stm32mp257f-ev1
 board"

This reverts commit 8f800b489e0db997359749bbdbfadec079318919.

On SoC "Secure Closed" the CPU has no access to the CoreSight
peripherals so the kernel pollutes the logs with several probe
deferred messages.

Revert the activation of CoreSight in board file.

Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: I51e86ec264a43e81b14d03e2dd06c3f0e080a254
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/427074
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 56 ----------------------
 1 file changed, 56 deletions(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index 66f981326c4f..0643948951e5 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -245,62 +245,6 @@ &cryp1 {
 	status = "okay";
 };
 
-&cs_cpu_debug0 {
-	status = "okay";
-};
-
-&cs_cpu_debug1 {
-	status = "okay";
-};
-
-&cs_cti0 {
-	status = "okay";
-};
-
-&cs_cti1 {
-	status = "okay";
-};
-
-&cs_cti_cpu0 {
-	status = "okay";
-};
-
-&cs_cti_cpu1 {
-	status = "okay";
-};
-
-&cs_etf {
-	status = "okay";
-};
-
-&cs_etm0 {
-	status = "okay";
-};
-
-&cs_etm1 {
-	status = "okay";
-};
-
-&cs_etr {
-	status = "okay";
-};
-
-&cs_funnel {
-	status = "okay";
-};
-
-&cs_replicator {
-	status = "okay";
-};
-
-&cs_stm {
-	status = "okay";
-};
-
-&cs_tpiu {
-	status = "okay";
-};
-
 &csi {
 	vdd-supply =  <&scmi_vddcore>;
 	vdda18-supply = <&scmi_v1v8>;
-- 
2.50.1

